easybuild/easyconfigs/m/Mesa/Mesa-12.0.2-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,64 @@
+# the purpose of the easyconfig is to build a Mesa for software rendering,
+# not hardware rendering. This means you want at least SSE4.2. We build:
+# - llvmpipe: the high-performance Gallium LLVM driver
+# - swr: Intel's OpenSWR
+# it will try to use the llvmpipe by default. It you want swr, do:
+# GALLIUM_DRIVER=swr
+
+easyblock = 'ConfigureMake'
+
+name = 'Mesa'
+version = '12.0.2'
+
+homepage = 'http://www.mesa3d.org/'
+description = """Mesa is an open-source implementation of the OpenGL specification -
+ a system for rendering interactive 3D graphics."""
+
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+# swr detects and builds parts specific for AVX and AVX2. If we use
+# -xHost, this always gets overwritten and will fail.
+toolchainopts = {'optarch': False}
+
+sources = [SOURCELOWER_TAR_XZ]
+source_urls = [
+    'https://mesa.freedesktop.org/archive/%(version)s',
+    'ftp://ftp.freedesktop.org/pub/mesa/%(version)s',
+    'ftp://ftp.freedesktop.org/pub/mesa/older-versions/%(version_major)s.x/%(version)s',
+]
+
+checksums = ['d957a5cc371dcd7ff2aa0d87492f263aece46f79352f4520039b58b1f32552cb']
+
+builddependencies = [
+    ('flex', '2.6.0'),
+    ('Bison', '3.0.4'),
+    ('Autotools', '20150215'),
+    ('pkg-config', '0.29.1'),
+    ('Mako', '1.0.4', '-Python-2.7.12'),
+    ('libxml2', '2.9.4'),
+]
+
+dependencies = [
+    ('zlib', '1.2.8'),
+    ('nettle', '3.2'),
+    ('libdrm', '2.4.70'),
+    ('LLVM', '3.9.0'),
+    ('X11', '20160819'),
+]
+
+# GLU is not part anymore of Mesa package!
+configopts = " --disable-osmesa --enable-gallium-osmesa --enable-gallium-llvm --enable-glx --disable-dri"
+configopts += " --disable-driglx-direct --with-gallium-drivers='swrast' --disable-egl"
```

Comment:
existing easyconfigs have ` --with-gallium-drivers='swrast,swr'`, did you change that deliberately?